src/view/jsView.js

```diff
@@ -5,23 +5,29 @@
 
 import { diceValueKey } from "../app";
 
-export const jsRenderView = (diceMap, elem) => {
+const template = document.createElement("template");
+
+template.innerHTML = `
```

Comment:
Why not just update the HTML file?

Reply:
we've been down that path before, and I think the desire was to keep everything in the JS file and rely less on "magic" in the HTML file.

Also, for React and View examples, the extra DOM in the HTML file won't help.